docs/audio.rst

```diff
@@ -3,8 +3,11 @@ Audio
 
 .. py:module:: audio
 
-This module allows you play sounds from a speaker attached to the Microbit.
-In order to use the audio module you will need to provide a sound source.
+This module allows you to play your own sounds. By default sound output
+will be via the edge connector on pin 0 and the built-in speaker **V2**. You
+can connect a wired headphones or a speaker to pin 0 and GND on the edge
+connector to hear the sounds. In order to use the audio module you will
+need to provide a sound source.
 
 A sound source is an iterable (sequence, like list or tuple, or a generator) of
 frames, each of 32 samples.
```

Comment:
>In order to use the audio module you will need to provide a sound source.
> 
> A sound source is an iterable (sequence, like list or tuple, or a generator) of frames, each of 32 samples.
>
>The ``audio`` modules plays samples at the rate of 7812.5 samples per second, which means that it can reproduce frequencies up to 3.9kHz.

I would probably remove all this quoted above.

-----

Reasoining:

> In order to use the audio module you will need to provide a sound source.

This is obvious, and if the rest of the content below (explaining the source) is removed, doesn't make sense to keep it.

> A sound source is an iterable (sequence, like list or tuple, or a generator) of frames, each of 32 samples.

This is already covered in the play() function description, and in the "Technical Details" section. For most users this is not applicable as they'll be using the `Sound` instances, which have nothing to do with `AudioFrames`.

>The ``audio`` modules plays samples at the rate of 7812.5 samples per second, which means that it can reproduce frequencies up to 3.9kHz.

Having this at the top makes this module seem more complicated than it is, and it is explained in the "Technical Details" section.

Reply:
Agreed and done in https://github.com/bbcmicrobit/micropython/pull/692/commits/01331f3e9d0e8ea461e9fae271fe3eb62153802f